scripts/apply-hotfixes.sh

```diff
@@ -74,4 +74,17 @@ else
     exit 1
 fi;
 
+# Patching commits for Tolerating Null Sources on Projected Volumes
+# UPDATE: OpenAPI generator v4.3.0 has the context manager as a functionality. Cherry-picking just the tests for completeness.
```

Comment:
Thanks! Could you remove this line and add "TODO: remove this patch when we release v20 clients"? Otherwise LGTM

Reply:
done.